routes/_components/timeline/PinnedStatuses.html

```diff
@@ -1,4 +1,5 @@
-<div role="feed" aria-label="Pinned toots" class="pinned-statuses">
+<h1 class="sr-only">Pinned statuses</h1>
```

Comment:
This heading even appears when there are no pinned statuses in a profile. It's probably worth hiding this along with the div that is the container when there are no such statuses. Otherwise, the display will be a h1 with Pinned Statuses, followed by an H1 with regular statuses intro.

Reply:
Good catch! Will fix.